### PR TITLE
knative-serving removal migration guide

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- initial version of the migration guide

**Related issue(s)**

### CODE REVIEW

File: docs/migration-guides/1.16-1.17.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+```bash
+helm delete knative-serving -n knative-serving
+``` 
+>This requires helm version 3.x
+
+This command renders knative-serving inactive, but all user created configuration is still available in the cluster. 
+To also remove this configuration run:
+
+```bash
+k delete crd \
```
Comment (nachtmaar) [inline docs/migration-guides/1.16-1.17.md:27]:
```suggestion
kubectl delete crd \
```

Comment (k15r) [reply docs/migration-guides/1.16-1.17.md:27]: 😄

**Review by marcobebway — DISMISSED** [review]

File: docs/migration-guides/1.16-1.17.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Migration from v1.16 to v1.17
+
+## Eventing
+
+In Release 1.17 we remove the support for Knative-Serving from kyma. After the upgrade Knative-Serving is still
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:5]:
```suggestion
In Release 1.17, we remove the support for Knative-Serving from Kyma. After the upgrade Knative-Serving is still 
```

File: docs/migration-guides/1.16-1.17.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Migration from v1.16 to v1.17
+
+## Eventing
+
+In Release 1.17 we remove the support for Knative-Serving from kyma. After the upgrade Knative-Serving is still 
+installed and fully functional in the cluster, but can be removed easily in order to free resources. Removing 
+Knative-Serving will not affect supported kyma functionality.
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:7]:
```suggestion
Knative-Serving will not affect supported Kyma functionality.
```

File: docs/migration-guides/1.16-1.17.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+Before you remove Knative-Serving from the cluster make sure that you take a backup of any of your own resources that used
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:9]:
```suggestion
Before you remove Knative-Serving from the cluster, make sure that you create a backup of any of your own resources that used 
```

File: docs/migration-guides/1.16-1.17.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+In Release 1.17 we remove the support for Knative-Serving from kyma. After the upgrade Knative-Serving is still 
+installed and fully functional in the cluster, but can be removed easily in order to free resources. Removing 
+Knative-Serving will not affect supported kyma functionality.
+
+Before you remove Knative-Serving from the cluster make sure that you take a backup of any of your own resources that used 
+Knative-Serving. You can do this by running:
+
+```bash
+kubectl get serving -A -oyaml > <some-backup-file>
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:13]:
```suggestion
kubectl get serving -A -oyaml > <backup-file-name>
```

Maybe it's just me, but `some backup file` sounds a little too casual. Hopefully `backup-file-name` is also clear

File: docs/migration-guides/1.16-1.17.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+```
+
+To remove the `knative-serving` chart execute:
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:16]:
```suggestion
To remove the `knative-serving` chart, execute:
```

File: docs/migration-guides/1.16-1.17.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+```bash
+helm delete knative-serving -n knative-serving
+``` 
+>This requires helm version 3.x
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:21]:
```suggestion
>**NOTE:** This requires Helm version 3.x
```

File: docs/migration-guides/1.16-1.17.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+This command renders Knative-Serving inactive, but all user created configuration is still available in the cluster. 
+To also remove this configuration run:
```
Comment (alexandra-simeonova) [inline docs/migration-guides/1.16-1.17.md:24]:
```suggestion
This command renders Knative-Serving inactive, but all user-created configuration is still available in the cluster. 
To also remove user-created configuration, run:
```

**Review by alexandra-simeonova — APPROVED** [review]

File: docs/migration-guides/1.15-1.16.md [hunk 9]
```diff
@@ -8,3 +8,40 @@ All Istio-related objects in Kyma are updated, but the migration may be necessar
...
+
+To remove the `knative-serving` chart, execute:
+
+```bash
+helm delete knative-serving -n knative-serving
+``` 
+>**NOTE:** This requires Helm version 3.x
+
+This command renders Knative-Serving inactive, but all user-created configuration is still available in the cluster. 
+To also remove user-created configuration, run:
```
Comment (sayanh) [inline docs/migration-guides/1.15-1.16.md:33]: I think we can be more clear here by: 
```
To remove the CustomResourceDefinitions for knative-serving along with the respective CustomResources
```

**Review by sayanh — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**